fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransaction.java

```diff
@@ -837,6 +849,10 @@ public LocalDate getSubmittedOnDate() {
         return submittedOnDate;
     }
 
+    public void setReversalExternalId(String reversalExternalId) {
```

Comment:
I don't think we wanna allow to override the reversal external ID once its set, do we?
I feel the reverse method should be more than enough to provide the reversal external ID.

Reply:
Done! Removed